src/_includes/install/linux.md

```diff
@@ -0,0 +1,113 @@
+
+### Install using a Linux package manager {:.no_toc}
+
+You have two options to install the Dart SDK on Ubuntu or Debian:
+
+* Use the [apt-get](#install-using-the-apt-get-package-manager) command.
+* Download a [`.deb`](#install-from-a-debian-package) package
+  and run the `dpkg` command.
+
+### Install using the `apt-get` package manager {:.no_toc}
+
+To install Dart with `apt-get`, perform the following steps.
+You need steps 1 to 3 only for the first install.
+
+1. Update the package index files and install the secure HTTP package.
+
+   ```console
+   $ sudo apt-get update && sudo apt-get install apt-transport-https
+   ```
+
+1. Download and add the Google Linux GPG public key.
+
+   ```console
+   $ wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub \
+     | sudo gpg  --dearmor -o /usr/share/keyrings/dart.gpg
+   ```
+
+1. Add the Dart package repository to your Linux system.
+
+   ```console
+   $ echo 'deb [signed-by=/usr/share/keyrings/dart.gpg arch=amd64] https://storage.googleapis.com/download.dartlang.org/linux/debian stable main' \
+     | sudo tee /etc/apt/sources.list.d/dart_stable.list
+   ```
+
+1. Use the `apt-get` command with [`sudo`][sudo] privileges.
+
+   ```console
+   $ sudo apt-get update && sudo apt-get install dart
+   ```
+
+### Install from a Debian package {:.no_toc}
+
+To install the Dart SDK from a Debian package (`*.deb`).
+
+1. Download the Dart SDK [Debian package](#){:.debian-link-stable}.
+
+1. Install the `*.deb` package with `dpkg` with [`sudo`][sudo] privileges.
+
+   ```console
+   $ sudo dpkg -i dart_3.2.6-1_amd64.deb
+   ```
```

Comment:
Consider clarifying they should replace the example with the downloaded package's filename.